extension-manifest-v3/src/manifest.chromium.json

```diff
@@ -125,6 +125,11 @@
       "matches": [
         "<all_urls>"
       ]
+    },
```

Comment:
Is this only chromium feature? If not, all of the manifests should be updated.

Reply:
Fixed in [`0097ddf` (#1750)](https://github.com/ghostery/ghostery-extension/pull/1750/commits/0097ddf6d60ee01382125300b57ea596b7203bbf). Also, manifest changes were reverted to keep the build process simple.